package.json

```diff
@@ -35,9 +35,8 @@
   },
   "dependencies": {},
   "devDependencies": {
-    "@blackbaud/skyux": "2.30.0",
-    "@blackbaud/skyux-builder": "1.29.0",
-    "@skyux-sdk/builder-plugin-skyux": "1.0.0-rc.5",
-    "@skyux/list-builder-common": "^3.0.0-rc.1"
```

Comment:
Why did we remove `list-builder-common` ?

Reply:
@Blackbaud-AlexKingman I believe it was there due to not being in Sky 2 previously. It is now released via Sky so we will use the version coming from there.

Reply:
gothcya. forgot we were doing that.